app/src/main/java/de/danoeh/antennapod/menuhandler/FeedItemMenuHandler.java

```diff
@@ -186,11 +186,6 @@ public static boolean onMenuItemClicked(Context context, int menuItemId,
                     GpodnetPreferences.enqueueEpisodeAction(actionNew);
                 }
                 break;
-            case R.id.move_to_top_item:
-                DBWriter.moveQueueItemToTop(selectedItem.getId(), true);
-                return true;
-            case R.id.move_to_bottom_item:
-                DBWriter.moveQueueItemToBottom(selectedItem.getId(), true);
             case R.id.add_to_queue_item:
```

Comment:
Didn't even see that... switch looks far better than if else if else if else... but then you make mistakes